components/gitpod-protocol/src/workspace-instance.ts

```diff
@@ -72,6 +72,11 @@ export interface WorkspaceInstance {
 
 // WorkspaceInstanceStatus describes the current state of a workspace instance
 export interface WorkspaceInstanceStatus {
+    // version is the current version of the workspace instance status
+    // Note: consider this value opague. The only guarantee given is that it imposes
+    //       a partial order on status updates, i.e. a.version > b.version -> a newer than b.
+    version: number;
```

Comment:
This should be `version?` for backwards compatibility.

I'm fine with promoting at some point, but let's stick with optional for now to avoid `undefined` issues.

Reply:
/hold

to discuss this.

Reply:
done

Reply:
/hold cancel